lib/diagnostic.rb

```diff
@@ -95,8 +119,8 @@ def eat_meat(food)
 # Write your answer as a comment in the section below.
 
 ##
-# your answer, in comments, here
-#
+# Using mixins makes the code you write more semantic and easier to read
+# for future developers and future you.
```

Comment:
This is true. It's not one of the primary reasons why composition is preferred. 
One big reason is the problem of multiple inheritance. A class can mix in an arbitrary number of modules. Classes can inherit from other classes, and gain methods and properties that way; however, this is less flexible than using mixins, and harder to organize.